src/txmempool.cpp

```diff
@@ -1580,7 +1581,7 @@ int CTxMemPool::Expire(std::chrono::seconds time)
     setEntries toremove;
     while (it != mapTx.get<entry_time>().end() && it->GetTime() < time) {
         // locked txes do not expire until mined and have sufficient confirmations
-        if (llmq::quorumInstantSendManager->IsLocked(it->GetTx().GetHash())) {
+        if (m_isman && m_isman->IsLocked(it->GetTx().GetHash())) {
```

Comment:
@PastaPastaPasta @UdjinM6 should we interrupt instead logic of "expiration"?

such as:
```
if (!m_isman) return 0;
```

if txmempool is initialized before LLMQContext and this clean-up process will be triggered before `ConnectManager` [can it actually happen?] we may expire some instant-send-locked transactions which is not good

Reply:
Good point 👍  There should be an `assert(m_isman);` outside of the loop instead I think.

Reply:
Done, resolved in latest push!